### DISCUSSION TITLE
create separate(debug) branch

### DISCUSSION THREAD

**Discussion by pidjan** [Xed-Editor/Xed-Editor#48] [General]
I suggest creating a separate branch, so as not to clog the main branch with commits and if someone wants to build the project but it throws error because u're fixing some error/adding something new.

**Comment by RohitKushvaha01** [comment]
> I suggest creating a separate branch, so as not to clog the main branch with commits and if someone wants to build the project but it throws error because u're fixing some error/adding something new.

You are correct idk why didn't did it earlier 😅

**Reply by pidjan** [reply]
i advice this just because i see that you commit project using github actions, and if it has error, no one can build project unless they/you fix it and try to build again.
